crates/holochain/src/conductor/conductor/agent_key_operations.rs

```diff
@@ -0,0 +1,152 @@
+use holochain_types::deepkey_roundtrip_backward;
+
+use super::*;
+
+/// The result type of an agent key revocation for an app.
+pub type RevokeAgentKeyForAppResult = HashMap<CellId, ConductorApiResult<()>>;
+
+impl Conductor {
+    /// Revoke an agent's key pair for all cells of an app.
+    ///
+    /// Writes a `Delete` action to the source chain of all cells of the app, which renders them read-only.
+    /// If DPKI is installed as conductor service, the agent key will be revoked there too and becomes
```

Comment:
It would be great if we used "DPKI" or "Deepkey" consistently throughout ;-) Right now it seems they're sometimes used ambiguously.

Reply:
I use DPKI to refer to the conductor service, and Deepkey to refer to the DNA. So by that measure, I think this one is correct?

Reply:
Maybe rather "if the DPKI conductor service is installed"?

Reply:
Yep!